packs/dotnet/dotnet/dashboards/dotnet.json

```diff
@@ -0,0 +1,585 @@
+{
+  "name": "DotNet App",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "DotNet App Overview",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 5
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "\n![Transactions](https://snipboard.io/O7Lz09.jpg)\n"
```

Comment:
Will this picture be available forever?

Reply:
Thanks for reminding me!! I put the images in an s3 bucket now.

Reply:
@aswanson-nr @jpvajda how would you suggest we host these images? Would it make sense to host them in this repo?

Reply:
I'm not clear why we are posting images inside the json for a dashboard.  images for dashboard are supposed to be stored in /dashboards directory of a quickstart as a unqiue file, along with the JSON file.

see apache for an example. 

https://github.com/newrelic/newrelic-observability-packs/tree/main/packs/apache/dashboards

I personally don't think we want to let 3rd parties link these to their own S3 buckets as that's an really good way for a hacker to compromise someone's NR account.

Reply:
This is a tough one, since this is not a screenshot of the dashboard, but an image that is part of the dashboard. It would be possible to link to the raw content of this image if it was checked into the repository. Another option would be to provide some sort of hosting for images like this. For now, I think we will need to go with the former. Just toss the image in the `/dashboards` folder for this quickstart and set the url for the image to `https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/main/packs/dotnet/dotnet/dashboards/07Lz09.jpg` (if the file name stays the same)

Reply:
Exactly @aswanson-nr, this is an image used **in** the dashboard, and it can make our dashboards look nicer.
@EmilHammarstrand can you follow Alec's suggestion and host it in the repo?

Reply:
Yes @polfliet - Check the latest commit 👍

Reply:
@EmilHammarstrand and @polfliet, looks good to me!

Reply:
@EmilHammarstrand Sorry! I made a mistake and forgot that the images in the `/dashboards` folder are limited to 6. 
Moving the 3 images to a `dotnet/images` folder should fix this issue. The links would change to 
* `https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/main/packs/dotnet/dotnet/images/Errors.jpg`
* `https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/main/packs/dotnet/dotnet/images/Transactions.jpg`
* `https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/main/packs/dotnet/dotnet/images/VM-metrics.jpg`

Reply:
@aswanson-nr No problem! I just commited the updated code, have a nice weekend everyone! 🎉